bridge/Mixer.cpp

```diff
@@ -101,9 +101,9 @@ bool waitForPendingJobs(const uint32_t timeoutMs, const uint32_t pollIntervalMs,
     while (totalSleepTimeMs < timeoutMs && transport.hasPendingJobs())
     {
         totalSleepTimeMs += pollIntervalMs;
-        transport.stop();
```

Comment:
Good change. I am not sure it is safe to stop multiple times or a small change could cause it to be problematic.